### ISSUE TITLE
Feature Request: Authorization with both Session Token and Login

### ISSUE THREAD

**Issue by Alb11747** [terry3041/pyChatGPT#48]
Is it possible to add a feature where if both a Session token and Login is provided use the session first, and if the token is either expired or invalid, fallback to login. This should be able to be done by just setting the session token and checking if the login screen appears.

Also a method to get the session token be added?
Currently using this:
```
config = chatgpt_get_config()

cookies_lst = chatgpt_instance.driver.execute_cdp_cmd(
    "Network.getCookies", {"urls": ["https://chat.openai.com/chat"]}
)["cookies"]

for cookie in cookies_lst:
    if cookie["name"] == "__Secure-next-auth.session-token":
        if cookie["value"] == config.get("session_token"):
            break
        logger.info("ChatBot - Saving session token")
        config["session_token"] = cookie["value"]
        chatgpt_set_config(config)
        break
else:
    logger.warning("ChatBot - Could not find session token")
```

Also, it would also be nice if there was a way to save the cloudflare token as well, but this isn't as necessary since logging in takes a lot longer.

**Comment by DevRoss** [comment]
good ideas and I have implemented this feature. #42

**Comment by terry3041** [comment]
Implemented in v0.3.9.5. Thanks @DevRoss.
